### ISSUE TITLE
[BUG]: Depends on vulnerable version of puppeteer

### ISSUE THREAD

**Issue by poolcvs** [IBMa/equal-access#1959]
### Project

extension-checker, accessibility-checker for Node

### Operating system

MacOS, Windows

### Description

Expect no vulnerablilities reported on “npm update”. Got:

    accessibility-checker  3.1.24-rc.0 - 3.1.73
    Depends on vulnerable versions of puppeteer
    node_modules/accessibility-checker

### Steps to reproduce

Run “npm update” with latest version of accessibility-checker as a dependency.

**Comment by shunguoy** [comment]
this should have been fixed with this pull request: https://github.com/IBMa/equal-access/pull/1960
the puppeteer was updated to the latest.

**Comment by MHoov** [comment]
Waiting for #1960 to be approved and moved to production

**Comment by shunguoy** [comment]
The update of the Puppeteer version requires a large change in the code base, which will be done in the coming version 4 release.
